packages/number-field/test/number-field.test.ts

```diff
@@ -1752,40 +1744,4 @@ describe('NumberField', () => {
             await clickBySelector(el, '.step-down');
         });
     });
```

Comment:
We should probably have tests for the following:

1. That the buttons don't appear in the a11ySnapshot
2. That they cannot receove keyboard focus